src/waypoint.jsx

```diff
@@ -32,6 +45,20 @@ function debugLog() {
   console.log(arguments); // eslint-disable-line no-console
 }
 
+function parseOffsetAsPixels(str) {
```

Comment:
Oh, I thought you were talking about memoizing the value. I was thinking we could store the computed value in an instance variable and only call the method(s) once. But I realize now that we can't do that because of the `contextHeight` being variable. Sorry for the confusion.